tests/restarting/to_7.1.0/CycleTestRestart-1.toml

```diff
@@ -4,6 +4,7 @@ maxTLogVersion = 6
 disableTss = true
 disableHostname = true
 disableEncryption = true
+allowDefaultTenant = false
```

Comment:
Why does this test need to disable the default tenant?

Reply:
I think this may have been leftover from debugging some noisy tests, let me see if there's still an issue with this one

Reply:
Ok, I remember now: I couldn't reproduce these errors because I didn't use the old binaries (These restarting tests are `to_7.1.0`) Is that version meant to support default tenants? The error in question is `CycleBadRead` in the restarting half of the tests

Reply:
I guess the scenario that won't be supported is if you try to downgrade to a version before what's on main, since that version won't know how to set up the test. Maybe that means we should split this test up into 2 versions, one that runs downgrades to older versions and another that runs downgrades to newer versions. The newer ones could allow default tenants while the older ones wouldn't.

Reply:
I think that happens when we advance FDB versions. I know the existing restarting tests get some versions bumped and new folders get created, but I'm not sure how to set that up if we're staying on the same version as main

Reply:
I think the new folders/tests get created as the need arises, such as in this case. On main, you would just write this using what the next minor version would be (7.2.0).